beacon-chain/sync/subscriber_beacon_blocks.go

```diff
@@ -55,6 +58,78 @@ func (s *Service) beaconBlockSubscriber(ctx context.Context, msg proto.Message)
 	return err
 }
 
+// reconstructAndBroadcastBlobs processes and broadcasts blob sidecars for a given beacon block.
+// This function reconstructs the blob sidecars from the EL using the block's KZG commitments,
+// broadcasts the reconstructed blobs over P2P, and saves them into the blob storage.
+func (s *Service) reconstructAndBroadcastBlobs(ctx context.Context, block interfaces.ReadOnlySignedBeaconBlock) {
+	startTime, err := slots.ToTime(uint64(s.cfg.chain.GenesisTime().Unix()), block.Block().Slot())
+	if err != nil {
+		log.WithError(err).Error("Failed to convert slot to time")
```

Comment:
why do we not return here if it errors?

Reply:
It's only used for log, and it doesn't feel right to ditch the rest of the process if we can't log one field correctly